apps/namadillo/src/hooks/useKeychainVersion.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import { useNamadaKeychain } from "./useNamadaKeychain";
+
+export const useKeychainVersion = (): string | undefined => {
+  const keychain = useNamadaKeychain();
+  const [version, setVersion] = useState<string>();
+
+  useEffect(() => {
+    (async () => {
+      const namadaKeychain = await keychain.namadaKeychain.get();
```

Comment:
Thinking loud here. Literally the `keychain` here is only `new NamadaKeychain()`, and the whole hook idea is not necessary. It was the reason that created the other issue on useEffect dependencies. Maybe we should just the `new NamadaKeychain()` directly